core/src/test/java/org/elasticsearch/index/query/RandomQueryBuilder.java

```diff
@@ -21,6 +21,7 @@
 
 import com.carrotsearch.randomizedtesting.generators.RandomInts;
 import com.carrotsearch.randomizedtesting.generators.RandomStrings;
+import org.apache.lucene.search.GeoPointDistanceQuery;
```

Comment:
I can see no reference to `GeoPointDistanceQuery` in this file. Is this import needed?